src/components/common/view/BorderedBox.js

```diff
@@ -130,12 +130,21 @@ const BorderedBox = ({
         <View style={[styles.boxCopyIconWrapper, showCopyIcon ? null : styles.boxCopyButtonWrapper]}>
           {showCopyIcon ? (
             <>
-              <RoundIconButton
-                onPress={copyToClipboard}
-                iconSize={22}
-                iconName="copy"
-                style={styles.copyIconContainer}
-              />
+              {enableIndicateAction && performed ? (
```

Comment:
@julianpolcode 
1. hmm I just wanted it to have normal background. not switch into success icon.
also we have a component ButtonWithDoneState that already does this.
2. also in the current export wallet page i dont see the "copy" icon anymore, so i'm not sure this fix is relevant
3. attach screenshots for visual fixes

@serdiukov-o-nordwhale also the whole layout of export wallet seems broken to me. the checkmark icon is at the left and not top, and there's no longer a "copy" icon but text "copy X", was there a design change?

Reply:
I updated the UI as requested

This is what the button looks like before being pressed
![inactive](https://user-images.githubusercontent.com/67687265/123387928-9535ac00-d598-11eb-8b82-8a34bb311d6e.png)

and after being pressed
![active](https://user-images.githubusercontent.com/67687265/123388048-ae3e5d00-d598-11eb-8c07-678609328fb1.png)